lib/screens/stock_take_details/sales/nozzle_tabs/sales_tab.dart

```diff
@@ -80,35 +75,69 @@ class _SalesPageState extends State<SalesPage>
                     }).toList(),
                     onChanged: (String? value) {
                       setState(() {
-                        defaultItem = value!;
+                        dropdownValue = value!;
                       });
                     },
                   ),
-                  TextField(
-                    decoration: InputDecoration(
-                      hintText: 'Volume',
-                      hintStyle: TextStyle(
-                        color: Colors.black38,
-                        fontSize: getProportionateScreenHeight(15),
-                      ),
-                      border: const UnderlineInputBorder(),
-                    ),
+                  getField(
+                    'Volume',
                   ),
                   SizedBox(height: getProportionateScreenHeight(5)),
-                  TextField(
-                    decoration: InputDecoration(
-                      hintText: 'Unit Price',
-                      hintStyle: TextStyle(
-                        color: Colors.black38,
-                        fontSize: getProportionateScreenHeight(15),
-                      ),
-                      border: const UnderlineInputBorder(),
-                    ),
+                  getField(
+                    'Unit Price',
                   ),
+                  SizedBox(height: getProportionateScreenHeight(40)),
+                  getVolumeRemaining(
```

Comment:
Do not refactor the volume remaining field. Just put the Text widget there to avoid redundant implementation.